### ISSUE TITLE
Not working - Aubio is failing with "could not find RIFF header" message

### ISSUE THREAD

**Issue by adamjakab** [adamjakab/BeetsPluginBpmAnalyser#11]
Needs fixing! Issue described here: https://github.com/aubio/aubio/issues/116

My plugin version (output of `beet bpmanalyser -v`) is:

```text
bpmanalyser: Bpm Analyser(beets-bpmanalyser) plugin for Beets: v1.3.3)
```

**Comment by adamjakab** [comment]
As the it states in the above mentioned issue: "The default binary is built with only a simple WAV file reader." so there is no immediate solution for this. This issue cannot be fixed without either:

a) convert mp3/flac -> wav and then process the file with aubio
b) find another library which can process mp3/flac

**Comment by adamjakab** [comment]
Solved in release v1.3.5.
